app/controllers/docker_manager/application_controller.rb

```diff
@@ -7,10 +7,10 @@ class ApplicationController < ActionController::Base
     protect_from_forgery
 
     def handle_unverified_request
-      super
-      clear_current_user
-      render plain: "['BAD CSRF']", status: 403
+      unless is_api?
+        super
+        clear_current_user
+        render plain: "['BAD CSRF']", status: 403
+      end
     end
-
-  end
```

Comment:
It looks like you accidentally deleted this `end`?

Reply:
Ugh. Fixed with a force push.